jdaviz/configs/cubeviz/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -0,0 +1,89 @@
+import pytest
+from packaging.version import Version
+import numpy as np
+import astropy
+from astropy.nddata import NDDataArray, StdDevUncertainty
+from specutils import Spectrum1D
+from regions import CirclePixelRegion, PixCoord
+
+ASTROPY_LT_5_3_2 = Version(astropy.__version__) < Version('5.3.2')
```

Comment:
This version is not yet released, but will be required in order to pull in bugfixes from https://github.com/astropy/astropy/pull/15082.

Reply:
You could do this though (uglier but works):

```python
ASTROPY_LT_5_3_2 = Version(astropy.__version__) < Version('5.3.2.dev')
```